e2e-tests/gef/cypress/e2e/pages/dashboard-page.js

```diff
@@ -4,6 +4,9 @@ const dashboardPage = {
   gefSubmission: () => cy.get('[data-cy="scheme-gef"]'),
   internalRefName: () => cy.get('[data-cy="internal-ref"]'),
   dashboardHome: () => cy.get('[data-cy="dashboard"]'),
+  rowIndex: {
+    link: (AINdealId, index = 1) => cy.get(`[data-cy*="deal__link--index--${index} deal__link--${AINdealId}"]`),
+  },
 };
```

Comment:
this is pretty confusing - its not at all clear what `rowIndex` means

I think something like this is a bit clearer?

```suggestion
  row: (index, AINdealId) => ({
    link: () => cy.get(`[data-cy*="deal__link--index--${index} deal__link--${AINdealId}"]`),
  }),
```

Reply:
done as suggested.